File: math/Check_is_square/Java/PerfectSquare.java
import java.util.Scanner

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		double num = sc.nextInt();
		if(Math.sqrt(num) % 1 == 0){
			System.out.println("This is a perfect square");
		} else {
			System.out.println("This is not a perfect square");
		}
	}
}

File: math/Check_is_square/ruby/perfect_square.rb
def is_perfect_square (num = 0)

  rest = Math.sqrt(num) % 1 
	return rest == 0
end


print("Enter a number: ")
num = gets.chomp.to_i

if is_perfect_square (num)
  print("Is perfect square")
else 
  print("Isn't perfect square")
end
